Rubberduck.Parsing/Inspections/Resources/InspectionsUI.de.resx

```diff
@@ -772,10 +772,10 @@ Falls der Parameter 'null' sein kann, bitte dieses Auftreten ignorieren. 'null'
     <value>Zwei Deklarationen sind aktiv und haben denselben Namen. Dies bedeutet, dass nur eine von ihnen zur Verwendung verfügbar ist.</value>
   </data>
   <data name="StopKeywordInspectionMeta">
-    <value>The 'stop' keyword halts execution and brings up the debugger. Avoid its usage.</value>
+    <value>Das 'Stop' Schlüsselwort unterbricht die Ausführung und ruft den Debugger auf. Vermeiden Sie die Verwendung in verteiltem Code.</value>
   </data>
   <data name="EmptyForLoopBlockInspectionName">
-    <value>Empty For Loop</value>
+    <value>Leere 'For...Next' Schleife</value>
```

Comment:
'For...Next'-Schleife

Reply:
Sollte man hier gegebenenfalls direkt das UTF-8 Ellipsenzeichen (U+2026, …) verwenden?